PWG/EMCAL/EMCALtasks/AliEmcalCorrectionTask.cxx

```diff
@@ -908,7 +894,7 @@ void AliEmcalCorrectionTask::SetupContainer(const AliEmcalContainerUtils::InputO
     /*result = fYAMLConfig.GetProperty(inputObjectPropertiesPath, "defaultClusterEnergy", tempString, false);
     if (result) {
       // Need to get the enumeration
-      AliVCluster::VCluUserDefEnergy_t clusterEnergyType = fgkClusterEnergyTypeMap.at(tempString);
+      AliVCluster::VCluUserDefEnergy_t clusterEnergyType = AliClusterContainre::fgkClusterEnergyTypeMap.at(tempString);
```

Comment:
typo

Reply:
Good catch, thanks! I'll fix it in my next PR